ChangeLog.d/resumption_cid.txt

```diff
@@ -0,0 +1,5 @@
+Bugfix
+   * Fix server connection identifier setting for outgoing encrypted records
+     on DTLS 1.2 session resumption. After DTLS 1.2 session resumption with
+     connection identifier, the Mbed TLS client now sends properly the server
```

Comment:
(minor)
```suggestion
     connection identifier, the Mbed TLS client now properly sends the server
```

Reply:
agreed (hence "minor")

Reply:
Do we require a CI rerun if the change will contain just this rewording?

Reply:
Not really. There is presumably a backport forthcoming as well?